system_baseline/metrics.py

```diff
@@ -21,15 +21,43 @@
 )
 
 inventory_service_requests = Histogram(
-    "drift_inventory_service_requests", "inventory service call stats"
+    "baseline_inventory_service_requests", "inventory service call stats"
 )
 
 inventory_service_exceptions = Counter(
-    "drift_inventory_service_exceptions", "count of exceptions raised by inv service"
+    "baseline_inventory_service_exceptions", "count of exceptions raised by inv service"
 )
 
-systems_compared_no_sysprofile = Histogram(
-    "drift_systems_compared_no_sysprofile",
-    "count of systems without system profile" "compared in each request",
-    buckets=[2, 4, 8, 16, 32, 64, 128, 256],
+
+inventory_service_no_profile = Counter(
+    "baseline_inventory_service_no_profile",
+    "count of systems fetched without a profile",
+)
+
+baseline_count = Gauge(
+    "baseline_count", "count of total number of baselines", multiprocess_mode="max"
+)
+
+baseline_account_count = Gauge(
+    "baseline_account_count",
+    "count of total number of accounts with baselines",
+    multiprocess_mode="max",
+)
+
+baseline_account_count_ones = Gauge(
+    "baseline_account_count_ones",
+    "count of total number of accounts with less than ten baselines",
+    multiprocess_mode="max",
+)
+
+baseline_account_count_tens = Gauge(
+    "baseline_account_count_tens",
+    "count of total number of accounts with less than one hundred baselines",
```

Comment:
To avoid confusion, could we update to "accounts with more than ten and less than one hundred baselines"